include/params.hpp

```diff
@@ -6,6 +6,7 @@
 /// Class containing input parameters for the simulated system
 class Params {
   public:
+    int skellysim_trajectory_version;
```

Comment:
Can we get rid of this? Doesn't seem like we're really using it and could result in accidental mixed state with something mutable like this. Properties is better for derived rather than user-defined data if it's something that will actually be needed for internal logic

Reply:
Okay, I got rid of it, we only really write it out once for the file header. When we read it back in it is just part of the input_map that we use to decide how to unpack the data.